docs/source/get-started/suggested-models.md

```diff
@@ -0,0 +1,173 @@
+# Suggested Models
+
+In theory, Lumigator supports any model that is uploaded to the [Hugging Face Hub](https://huggingface.co/models?pipeline_tag=summarization&sort=trending)
+and is trained for *summarization*. However, we have extensively tested a few models and created
```

Comment:
suggestion: link to our blog post on model testing, but not a must

Reply:
Good idea.